packages/create-astro/src/index.ts

```diff
@@ -66,6 +76,25 @@ export async function main() {
     verbose: false,
   });
 
+  const selectedTemplate = TEMPLATES.find(template => template.value === options.template);
+  let renderers: string[] = [];
+  
+  if (selectedTemplate?.renderers === true) {
+    const result = /** @type {import('./types/internal').Options} */ await prompts([
+      {
+        type: 'multiselect',
```

Comment:
is this defaulting to none selected or all selected (or first selected?)

Reply:
Good question! None currently, but what do you think the best default is? None seems like the least clicks to enable just your preferred frameworks (rather than deselecting 3 other options, you select one)

Reply:
No preference from me.